### PR TITLE
update readthedoc requirements

### PR DESCRIPTION
**Pull request by samuelgarcia**

### CODE REVIEW

**Comment by samuelgarcia** [comment]
@alejoe91 : if you have have to merge this it would be cool. It mak ethe cache working again. to have fast test

File: spikeinterface/core/baserecording.py [hunk 1]
```diff
@@ -81,13 +81,22 @@ def get_traces(self,
...
                    ):
         segment_index = self._check_segment_index(segment_index)
         channel_indices = self.ids_to_indices(channel_ids, prefer_slice=True)
         rs = self._recording_segments[segment_index]
         traces = rs.get_traces(start_frame=start_frame, end_frame=end_frame, channel_indices=channel_indices)
         if order is not None:
             traces = np.asanyarray(traces, order=order)
+        if return_scaled:
+            gains = self.get_property('magnitude_gain')
+            offsets = self.get_property('magnitude_offset')
```
Comment (alejoe91) [inline spikeinterface/core/baserecording.py:94]: @samuelgarcia why not just `gain` and `offset`?

Comment (samuelgarcia) [reply spikeinterface/core/baserecording.py:94]: It is more clear like that and we will have not conflict with user property.
gain can be lot of things (amplifier gain, ...) here we could be even more clear with "gain_to_microvolt")

Comment (alejoe91) [reply spikeinterface/core/baserecording.py:94]: but then if we had `gain_to_uV` how would you call offset?

IMO as long as we are consistent and we document it well, `gain` and `offset` are good names

Comment (samuelgarcia) [reply spikeinterface/core/baserecording.py:94]: offset_to_uV. lets be consistent and sad.

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**